web/documentserver-example/php/index.php

```diff
@@ -74,12 +74,6 @@ function routers()
         $view->render();
         return;
     }
-    if (str_starts_with($path, '/assets')) {
-        $response = assets();
-        $response['status'] = 'success';
-        echo json_encode($response);
-        return;
-    }
```

Comment:
Because in PHP "assets" route is already taken. It leads to the assets folder.
In Ruby this causes an overriding error, but in PHP it simply does nothing.

> Why did you mark this route as wrong? What's wrong here?

Reply:
Have you tested this if we run the example in a container? I'm not sure this will work.

Reply:
> Have you tested this if we run the example in a container? I'm not sure this will work.

I have tested this in container. It's working. This route is trying to override "assets". This block of code WILL NOT WORK nowhere. Instead of `/assets?fileName=sample.docx`, the code now contains `/assets/document-templates/sample/sample.docx`. 
Conversation resolved.

Reply:
Please calm down and don't yell at me.

In the PHP example, we've [another endpoint](https://github.com/ONLYOFFICE/document-server-integration/blob/34e60cd33f98032954ef70ae4af5b61a4a8f3540/web/documentserver-example/php/index.php#L89) that performs the same tasks as `/assets`. It'd be beneficial to delete it and update all mentions to use the new implementation instead.

Reply:
> > Have you tested this if we run the example in a container? I'm not sure this will work.
> 
> I have tested this in container. It's working. This route is trying to override "assets". This block of code WILL NOT WORK nowhere. Instead of `/assets?fileName=sample.docx`, the code now contains `/assets/document-templates/sample/sample.docx`. Conversation resolved.

I additionally tested this in Docker and deployed the server on the host machine - everything works.

Reply:
> Have you tested this if we run the example in a container? I'm not sure this will work.

I additionally tested this in Docker and deployed the server on the host machine - everything works.

Reply:
@ZEROM22, thanks.